index.js

```diff
@@ -25,6 +25,7 @@ const defaultConfig = {
   autoPolyfill: false,
   inputPath: 'translations',
   outputPath: 'translations',
+  throwICUArgumentMismatch: false,
```

Comment:
`throwICUArgumentMismatch` -> `errorOnNamedArgumentMismatch`

(Context: I want to go back and rename `throwMissingTranslations` to `errorOnMissingTranslations`)